### DISCUSSION TITLE
RAC, SSR, Remix, Vite, and the future

### DISCUSSION THREAD

**Discussion by BMCwebdev** [adobe/react-spectrum#6136] [Q&A] [answered]
I have a question about using React Aria Components in a Remix app. It appears that if you want to use SSR in Remix, you must use Vite as the bundler. Please correct me if I am wrong. If so, are there plans to allow other bundlers in the future? Vite makes the most sense given that it is now Remix's default build, but this would just be helpful information to have. Thank you.

**Comment by snowystinger** [comment, marked as answer]
I don't think we do anything that prevents you from using any bundlers? What issues are you running into or are you worried about running into?

We have instructions for using our components with Remix, you can find more information searching in our documentation. We also have an example test app which may help you get set up https://github.com/adobe/react-spectrum/tree/main/examples/remix

**Reply by BMCwebdev** [reply]
Oh, I forgot about this one 😊
We ended up pivoting to Next for other reasons, but thank you very much!
